src/components/PageBanner/_PageBanner.scss

```diff
@@ -4,6 +4,7 @@
   display: flex;
   position: relative;
   height: 17rem;
+  margin-top: 47px;
```

Comment:
This seems oddly specific. Is this a way to push the page banner down below the top Menu bar?